rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -34,16 +34,16 @@ namespace rsx
 		rsx_log.trace("RSX method 0x%x (arg=0x%x)", reg << 2, arg);
 	}
 
-	template<bool FlushDMA, bool FlushPipe>
+	template<bool FlushDMA, bool FlushPipe, bool IsImmediate = false>
```

Comment:
This variable can be named something better to convey what it is actually doing. Just name it ForceUpdate or something.